python/ray/autoscaler/_private/updater.py

```diff
@@ -488,7 +491,7 @@ def do_update(self):
         with cli_logger.group(
             "Starting the Ray runtime", _numbered=("[]", 7, NUM_SETUP_STEPS)
         ):
-            global_event_system.execute_callback(CreateClusterEvent.start_ray_runtime)
+            global_event_system.execute_callback(CreateClusterEvent.start_ray_runtime, {"node_context": node_context})
```

Comment:
Do we have any corresponding events for node stop/termination?

Reply:
In the current autoscaler implementation, there aren't any callbacks executed for node stop/termination.